### ISSUE TITLE
Unable to start the otbr-agent on OpenWrt 21.02

### ISSUE THREAD

**Issue by Semi-Bin-Blain** [openthread/ot-br-posix#1042]
Hi there, I run the OTBR on the OpenWrt, but there is a problem when I updated the OpenThread to version 1.2, I can't start the otbr-agent normally as I used to. The log shows that the otbr-agent exited by the error code 5, there is some file not found when setup. I can't figure it out, so I come for help.

The logs:
```
otbr-agent[21120]: [INFO]-UTILS---: Running 0.3.0-1fce7d6-dirty
otbr-agent[21120]: [INFO]-UTILS---: Thread version: 1.2.0
otbr-agent[21120]: [INFO]-UTILS---: Thread interface: wpan0
otbr-agent[21120]: [INFO]-UTILS---: Backbone interface: 
otbr-agent[21120]: [INFO]-UTILS---: Radio URL: spinel+hdlc+uart:///dev/ttyACM0?uart-baudrate=115200
otbr-agent[21120]: [INFO]-PLAT----: RCP reset: RESET_POWER_ON
otbr-agent[21120]: [NOTE]-PLAT----: RCP API Version: 4
otbr-agent[21120]: [CRIT]-PLAT----: exit(5): SetUp line 177, open, No such file or directory
```

1. ot-br-posix commit id : 58fccc2594e39b864b5bfedf04e76bde47b5ace2 & e149a60eb1d3dfaadf23235aad8c9daea612d62d the latest version and the nordic suggested version
2. Thread Version 1.2
3. IEEE 802.15.4 hardware platform: NRF52840DK
4. RCP software: ncs1.7.0/mrf/samples/openthread/coprocessor
5. OpwnWrt Version: stable release 21.02

Thanks in advance.

**Comment by Semi-Bin-Blain** [comment]
We have solved this problem, the Openthread Commit [#6552](https://github.com/openthread/openthread/pull/6552/commits/864260fb4a9847c851c3132c41b05dabfe172198#:~:text=%23ifndef%20OPENTHREAD_POSIX_CONFIG_DAEMON_SOCKET_BASENAME-,%23ifndef%20OPENTHREAD_POSIX_CONFIG_DAEMON_SOCKET_BASENAME,%23endif,-/**) shows that the socket file and lock file has changed the location and basename, the code shows that the otbr-agent should find the file in the directory `/run`, the new location is the `/run` in Linux, in Openwrt, it is the `/var/run`;  so we create a soft link named `/run` point to the `/var/run`, then the otbr-agent can find the file in the correct location, and the Thread starts successfully.

ot-br-posix commit id : e149a60

**Comment by mrninhvn** [comment]
> Hi there, I run the OTBR on the OpenWrt, but there is a problem when I updated the OpenThread to version 1.2, I can't start the otbr-agent normally as I used to. The log shows that the otbr-agent exited by the error code 5, there is some file not found when setup. I can't figure it out, so I come for help.
> 
> The logs:
> 
> ```
> otbr-agent[21120]: [INFO]-UTILS---: Running 0.3.0-1fce7d6-dirty
> otbr-agent[21120]: [INFO]-UTILS---: Thread version: 1.2.0
> otbr-agent[21120]: [INFO]-UTILS---: Thread interface: wpan0
> otbr-agent[21120]: [INFO]-UTILS---: Backbone interface: 
> otbr-agent[21120]: [INFO]-UTILS---: Radio URL: spinel+hdlc+uart:///dev/ttyACM0?uart-baudrate=115200
> otbr-agent[21120]: [INFO]-PLAT----: RCP reset: RESET_POWER_ON
> otbr-agent[21120]: [NOTE]-PLAT----: RCP API Version: 4
> otbr-agent[21120]: [CRIT]-PLAT----: exit(5): SetUp line 177, open, No such file or directory
> ```
> 
> 1. ot-br-posix commit id : [58fccc2](https://github.com/openthread/ot-br-posix/commit/58fccc2594e39b864b5bfedf04e76bde47b5ace2) & [e149a60](https://github.com/openthread/ot-br-posix/commit/e149a60eb1d3dfaadf23235aad8c9daea612d62d) the latest version and the nordic suggested version
> 2. Thread Version 1.2
> 3. IEEE 802.15.4 hardware platform: NRF52840DK
> 4. RCP software: ncs1.7.0/mrf/samples/openthread/coprocessor
> 5. OpwnWrt Version: stable release 21.02
> 
> Thanks in advance.

Hi, I have the same setup and border router can't open `/dev/ttyACM` device.
Do you have any suggestions on how to fix it?
Thank you.
